smact/tests/test_utils.py

```
@@ -92,3 +93,73 @@
+
+    def test_oxidation_states_write(self):
+        threshold = 1000
+        filename = "test_ox_states"
+        filename_w_zero = "test_ox_states_w_zero"
+        comment = "Testing writing of ICSD 24 oxidation states list."
+        self.ox_filter.write(filename, threshold, comment=comment)
+        self.ox_filter.write(
+            filename_w_zero, threshold, include_zero=True, comment=comment
+        )
+        self.assertTrue(os.path.exists(f"{filename}.txt"))
+        assert [line for line in open(f"{filename}.txt")] == [
```

Ruff

Unnecessary `list` comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)